src/transformers/models/llama/convert_llama_weights_to_hf.py

```diff
@@ -301,6 +311,13 @@ def main():
         help="Location to write HF model and tokenizer",
     )
     parser.add_argument("--safe_serialization", type=bool, help="Whether or not to save using `safetensors`.")
+    parser.add_argument(
+        "--llama_version",
+        choices=[1, 2],
+        default=1,
+        type=int,
+        help="Version of the Llama model to use. Current supported versions are Llama1 and Llama2.",
```

Comment:
let's explain a bit why this is needed!

Reply:
Sorry I will change back the default from 4096 to 2048 (forgot I had changed this in the first commit) for backwards compatibility. I will also add a comment for the `llama_version` argument.